Core/BookmarksCoreDataStorage.swift

```diff
@@ -714,6 +904,21 @@ extension BookmarksCoreDataStorage {
     }
 }
 
+// MARK: Constants
+extension BookmarksCoreDataStorage {
+    struct Constants {
```

Comment:
worth changing this to enum while you're here?

Reply:
👍 have made that change 🙂